adhocracy4/polls/assets/PollQuestions.jsx

```diff
@@ -38,6 +38,8 @@ class PollQuestions extends React.Component {
       loadingPage: true
     }
 
+    this.handleTermsOfUse = this.handleTermsOfUse.bind(this)
```

Comment:
maybe this can be omitted if you write it as arrow function expression on line 194?

Reply:
hmm, I'd keep it like this till we rewrite it in functional as we do the same in the other components